src/protocol/mod.rs

```diff
@@ -1023,6 +1037,51 @@ mod load {
         }
     }
 
+    mod regex_matching {
+        use super::*;
+        use pretty_assertions::assert_eq;
+
+        #[test]
+        fn parses_a_regex_command_matcher() -> R<()> {
+            let step = test_parse_one(
+                r"
+                    |protocols:
+                    |  - protocol:
+                    |      - regex: \d
+                ",
+            )?
+            .steps[0]
+                .clone();
```

Comment:
I think you could do `.get(0).unwrap()`.

Reply:
i had that but clippy yelled at me